File: src/VRE.Vridge.API.Client/Messages/v1/Control/Requests/RequestEndpoint.cs
namespace VRE.Vridge.API.Client.Messages.v1.Control.Requests
{
    public class RequestEndpoint : ControlRequestHeader
    {        
        public string RequestedEndpointName;

        public RequestEndpoint(string name)
        {
            Code = (int) ControlRequestCode.RequestEndpoint;
            RequestedEndpointName = name;
        }
    }
}

File: src/java/VRE.Vridge.API/VRE.Vridge.API.Client/src/main/java/com/riftcat/vridge/api/client/java/control/requests/RequestEndpoint.java
package com.riftcat.vridge.api.client.java.control.requests;

import com.riftcat.vridge.api.client.java.control.ControlRequestCode;

public class RequestEndpoint extends ControlRequestHeader{

    public String RequestedEndpointName;

    public RequestEndpoint(String name, String appName){
        super(appName);
        Code = ControlRequestCode.RequestEndpoint;
        RequestedEndpointName = name;
    }

}
